### PR TITLE
Fix #113: cohort pings text color should match cohort

### LINKED ISSUE #113: Match per-metric count text color w graph cohort colors

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

File: src/components/views/Metric.js [hunk 1]
```diff
@@ -16,13 +16,12 @@ export default withRouter(props => {
...
-            let itemColor = {backgroundColor: `rgb(${props.populationColors[i][0]}, ${props.populationColors[i][1]}, ${props.populationColors[i][2]})`};
-            let borderColor = {border: `1px solid rgb(${props.populationColors[i][0]}, ${props.populationColors[i][1]}, ${props.populationColors[i][2]})`};
+            let color = `rgb(${props.populationColors[i][0]}, ${props.populationColors[i][1]}, ${props.populationColors[i][2]})`;
 
             pairs.push(
                 <div key={populationName}>
-                    <dt style={itemColor}>{populationName}</dt>
-                    <dd style={borderColor}>
+                    <dt style={{backgroundColor: color}}>{populationName}</dt>
+                    <dd style={{border: `1px solid ${color}`, color}}>
```
Comment (openjck) [inline src/components/views/Metric.js]: The `, color` thing confused me for a second. Clever.

**Comment by openjck** [comment]
I spoke with Rob about this in #113. He said he was looking for the names and/or numbers at the top of the page to be colored, not the numbers in the tooltips.

But hopefully there's not too much rework needed to make that happen.

**Comment by spasovski** [comment]
I might leave this as is anyway since it's a neat effect but I'll add another commit that addressed the original intent.

**Comment by spasovski** [comment]
updated

**Comment by openjck** [comment]
Nice!

**Pull request merged**